pages/database.tsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import useSWR from "swr";
```

Comment:
we'll be using fetch requests to fetch our data so let's use async fetch requests instead of swr for consistency! Try referencing what Sydney did in her [playerProfile page](https://github.com/calblueprint/ogsc/blob/7260e6f738566a7db7ad4ccdb09542245cf1396b/pages/player/%5Bid%5D.tsx#L20) for reference. You should do follow a similar format in which you create a state variable to represent all your users and populate it with the users table using the fetch request in useEffect